docs/user-guides/cluster-issuers.md

```diff
@@ -0,0 +1,64 @@
+# Creating cluster wide issuers
+
+`cert-manager` has the concept of `ClusterIssuers`. These are a non-namespaced and cluster-scoped version of an `Issuer`. The specification of a `ClusterIssuer` is exactly the same as that of an `Issuer`, but there are a couple of nuances you need to be aware of.
+
+```yaml
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: ca-cluster-issuer
+spec:
+  ca:
+    secretName: ca-key-pair
+```
+
+```yaml
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging-cluster-issuer
+spec:
+  acme:
+    server: https://acme-staging.api.letsencrypt.org/directory
+    email: user@example.com
+    privateKeySecretRef:
+      name: letsencrypt-staging
+    http01: {}
+```
+
+The two manifests above define two `ClusterIssuers`. As with `Issuer` resources we reference `Secrets` necessary for configuration. In order to configure which namespace these `Secrets` will be in, we need to specify the `--cluster-resource-namespace` flag on the `cert-manager` controller. By default, this flag will be set to `kube-system`.
```

Comment:
`As ClusterIssuer resources do not specify a namespace, we must configure a namespace that cert-manager will use to store supporting resources required for each ClusterIssuer` (or words to that effect).